iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/group/members/group.members.component.html

```diff
@@ -22,16 +22,16 @@
 
 		<div class="row iam-div-row"
```

Comment:
I know that we're putting effort on the new dashboard but these changes should not be so difficult to add:

I would add a div visibile for non privileged users (not admin not group manager not reader) with a "Not enough privileges" message.
Then I'd move the ng-if=" is admin or is reader or is group manager" inside the other div and a finer grained authz for buttons (only for admins and group managers) for the delete button for example

Does this make sense?